packages/b/blake3/xmake.lua

```diff
@@ -0,0 +1,48 @@
+package("blake3")
+    set_homepage("https://blake3.io/")
+    set_description("BLAKE3 is a cryptographic hash function that is much faster than MD5, SHA-1, SHA-2, SHA-3, and BLAKE2; secure, unlike MD5 and SHA-1 (and secure against length extension, unlike SHA-2); highly parallelizable across any number of threads and SIMD lanes, because it's a Merkle tree on the inside; capable of verified streaming and incremental updates (Merkle tree); a PRF, MAC, KDF, and XOF, as well as a regular hash; and is a single algorithm with no variants, fast on x86-64 and also on smaller architectures.")
+    set_license("CC0-1.0")
+
+    add_urls("https://github.com/BLAKE3-team/BLAKE3/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/BLAKE3-team/BLAKE3.git")
+    add_versions("1.3.1", "112becf0983b5c83efff07f20b458f2dbcdbd768fd46502e7ddd831b83550109")
+
+    on_install(function (package)
+        local configs = {}
+        io.writefile("xmake.lua", [[
+            add_rules("mode.release", "mode.debug")
+            target("blake3")
+                set_kind("$(kind)")
+                add_files("c/blake3.c", "c/blake3_dispatch.c", "c/blake3_portable.c")
+                add_headerfiles("c/blake3.h")
+
+                if is_arch("x86_64") then
```

Comment:
if is_arch("x86_64", "x64") then

Reply:
Gotcha, am applying all of your changes. For my own learning though, what is the difference between x86_64 and x64 supposed to be?

Reply:
x86 x64 for windows/msvc